### PR TITLE
Adding Auth Route

### PR DESCRIPTION
**Pull request by Lalit3716**
Adding Login/Signup Page

Will close issue #22

### CODE REVIEW

File: client/src/components/AuthForm/index.tsx [hunk 1]
```diff
@@ -82,25 +81,6 @@ const AuthForm: React.FC = () => {
...
-              <StyledTextField
-                {...field}
-                placeholder="Enter your username"
-                error={Boolean(errors.username)}
-                helperText={errors.username && errors.username.message}
-              />
-            )}
-          />
-        </Stack>
-      )}
```
Comment (Lalit3716) [inline client/src/components/AuthForm/index.tsx]: Why we removed this piece of code ?

Comment (Rishabh-malhotraa) [reply client/src/components/AuthForm/index.tsx]: I don't think we require a username, an email-id should be enough

**Comment by Rishabh-malhotraa** [comment]
LGTM! Thank you @Lalit3716 
image

File: client/src/components/AuthForm/index.tsx (hunk 1 above)
Comment (Lalit3716) [reply client/src/components/AuthForm/index.tsx]: Oh okay cool!

**Pull request merged**